app-frontend/src/app/pages/library/projects/detail/projectScenes/projectScenes.html

```diff
@@ -14,57 +14,54 @@ <h1 class="h3 page-title">
                 ng-show="!$ctrl.$parent.selectedScenes.size && !$ctrl.isEditingProjectName && !$ctrl.isSavingProjectNameEdit">
           <i class="icon-pencil"></i>
         </button>
-        <div class="color-danger" ng-show="$ctrl.showError">
+        <div class="color-danger" ng-if="$ctrl.showError">
           There was an error renaming the project.
           <button class="btn btn-tiny" type="submit"
                   ng-click="$ctrl.dismissErrorMessage()">
             <i class="icon-cross color-danger"></i>
           </button>
         </div>
-        <div ng-show="$ctrl.isEditingProjectName">
+        <div ng-if="$ctrl.isEditingProjectName">
```

Comment:
Yeah I think we should use `ng-if` wherever possible! It just comes with the understanding that if you `ng-if` a component the component is actually destroyed and initialized with each corresponding change in the `ng-if` condition. I don't know if this would actually cause a problem, but it certainly can give us the benefit of not initializing unseen components.